libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -379,6 +387,16 @@ bool AC_WPNav::advance_wp_target_along_track(float dt)
         reached_leash_limit = true;
     }
 
+    /*
+      also check for crosstrack limit, allowing us to constrain leash
+      point to within a given distance of the track. This can be used
+      to prevent high wind from causing a blowout of the distance from
+      the desired track
+     */
+    if (_crosstrack_max > 0 && _track_error_xy*0.01 > _crosstrack_max) {
```

Comment:
@tridge does the `_crosstrack_max > 0` require the use of machine epsilon in the comparison? I see that it's defined as `AP_Float` -- how is that different from vanilla `float`?

Reply:
using > 0 here just means it has been set at all. We could use is_positive(), but not really needed
AP_Float is a float that is a parameter in AP_Param